dist/vue-carousel.min.js

```diff
@@ -1,6 +1,1482 @@
\ No newline at end of file
+!(function(t, e) {
```

Comment:
Is formatting of minified distribution files intended?

Reply:
^^^ good catch! Doesn't seem very minified if we format lol